docs/decisions/0003-reduce-bokchoy-testing.rst

```diff
@@ -0,0 +1,67 @@
+Status
+======
+Draft
+
+
+Context
+=======
+
+edx-platform bokchoy tests are slow, flaky and difficult to debug.  A quick assessment of their value shows that they might be more trouble than they are worth.  And that we might get the same benefit with far fewer tests.
+
+Baseline Data:
+--------------
+
+    This data was collected based on the results of bokchoy tests run across all edx-platform PRs over the last 7 days.
+
+    * Total number of builds: 253(across 106 PRs)
+    * Failures: 49(across 24 PRs)
+        * True Failures: 10(across 6 PRs)
+        * Failures that wouldn’t be caught by other test: 3(on 1 PR)
```

Comment:
Remove initial indents, add a blank line before the 2 sub-bullets, and add only a 2-space indent before the 2 sub-bullets.